policyengine_us/tests/policy/baseline/gov/states/hi/tax/income/alternative_tax/hi_alternative_tax_on_capital_gains_eligible.yaml

```diff
@@ -0,0 +1,45 @@
+- name: Hawaii eligible alternative tax on capital gains for Single
+  period: 2021
+  input:
+    state_code: HI
+    filing_status: SINGLE
+    taxable_income: 25_000
+  output:
+    hi_alternative_tax_on_capital_gains_eligible: true
+
+- name: Hawaii eligible alternative tax on capital gains for arried filing separately
+  period: 2021
+  input:
+    state_code: HI
+    filing_status: SEPARATE
+    taxable_income: 23_000
+  output:
+    hi_alternative_tax_on_capital_gains_eligible: false
+
+- name: Hawaii eligible alternative tax on capital gains for head of household
+  period: 2021
+  input:
+    state_code: HI
+    filing_status: HEAD_OF_HOUSEHOLD
+    taxable_income: 35_000
+  output:
+    hi_alternative_tax_on_capital_gains_eligible: false
+
+- name: Hawaii eligible alternative tax on capital gains for married filing jointly
+  period: 2021
+  input:
+    state_code: HI
+    filing_status: JOINT
+    taxable_income: 35_000
+  output:
+    hi_alternative_tax_on_capital_gains_eligible: false
+
+- name: Hawaii eligible alternative tax on capital gains for widow
```

Comment:
copy the first test but change the period to 2023 to show that under the same circumstances, in 2023, the tax unit is not eligible